src/axom/primal/geometry/Plane.hpp

```diff
@@ -108,29 +108,42 @@ class Plane
   /// \name Constructors
   /// @{
 
+  /*!
+   * \brief Constructs a Plane with a zero normal vector and zero offset
+   *
+   * \post isValid() == false
+   */
+  Plane() = default;
+
   /*!
    * \brief Constructs a Plane with a given normal, \f$ \mathcal{N} \f$, that
    *  passes through the specified point, \f$  x \f$
    *
    * \param [in] normal the supplied plane normal, \f$ \mathcal{N} \f$
    * \param [in] x a specified point that the plane passes through, \f$ x \f$
+   * \param [in] normalize if true, normalize the given vector (default: true)
    *
-   * \note The supplied normal will be normalized, such that, the Plane's normal
-   *  will always be a unit normal.
+   * \note The Plane's normal should always be a unit normal, so the
+   *  normalize flag should only be set to false when the supplied
+   *  normal has already been normalized.
```

Comment:
> The Plane's normal should always be a unit normal, so the normalize flag should only be set to false when the supplied normal has already been normalized

This might be a good time to question whether a plane's normal should always be normalized. I don't recall why that was the case, but w/ the new parameter, I'd suggest the default should be `false` and a user can normalize if desired.

Reply:
Specifically, I'd think the if the normal vector for a plane is $({10,0,0})$, then normalizing it $({1,0,0})$ will change the `m_offset`, but both would be valid planes.

Reply:
The blame commit history shows the reasoning behind why normals were always normalized: 
https://github.com/LLNL/axom/commit/74d1cb26c9a9767272c9464f86120383e0a0fd4f

It looks like the intention back then was to make the normalization optional.
Sounds like the option was removed and made mandatory to resolve issues with ambiguous constructors in the past, back when the Plane constructors used to represent `Vector` and `Point` with pointers.

Reply:
Thanks @bmhan12 for digging into the history. 
I'm reading it slightly differently though:

>    * \note The Plane constructor will always normalize the supplied normal
   *  unless the caller indicates that the supplied normal is a unit normal
   *  via the third (optional) argument to the constructor.

Sounds like the expectation was for the normal to be a unit vector, but to avoid the extra `normalize` call when the user claims the provided normal is already a unit vector.

My preference would be to not require the normal vector to be a unit vector -- i.e. the normalization should be opt-in. And a plane should be valid as long as the normal is not the zero vector.

Reply:
I updated the documentation to say that the normal is not required to be a unit vector.